actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -135,9 +135,8 @@ def formats=(extensions)
         }
       end
 
-      # Receives an array of mimes and return the first user sent mime that
+      # Receives an array of MIMEs and return the first user sent MIME that
```

Comment:
If we're going to be pedantic these should be `MIME types` and `MIME type` - `MIME` is the group of RFCs that make up the standard